packages/frontity-org-theme/src/components/dropdown.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { styled, css, connect } from "frontity";
+import { Connect } from "frontity/types";
+import FrontityOrg from "../../types";
+import Arrow from "./icons/arrow";
+import { addAlpha } from "../utils";
+
+const Dropdown: React.FC<Connect<
+  FrontityOrg,
+  {
+    className?: string;
+    css?: ReturnType<typeof css>;
+  }
+>> = ({ className, css, children, state }) => {
+  // Define the inner state.
+  const [isOpen, setOpen] = React.useState(false);
+
+  // Get rendered `button` and `content` from `children`.
+  const [button, content] = React.Children.toArray(children);
+
+  // Get colors from the state.
+  const buttonColor = state.theme.colors.frontity;
+  const separatorColor = addAlpha(state.theme.colors.primary, 0.08);
+
+  return (
+    /**
+     * Pass down `className` and `css` props to keep styles
+     * added by processors and classes.
+     */
+    <Container className={className} css={css}>
```

Comment:
I don't think that adding `css={css}` is necessary, is it?

From their docs (https://emotion.sh/docs/css-prop#use-the-css-prop):
> Any component or element that accepts a className prop can also use the css prop. The styles supplied to the css prop are evaluated and the computed class name is applied to the className prop.

Could you please check it out and confirm either if it's needed or if adding `className` is enough? Thanks! :D

Reply:
Is it enough then? Did you check that it works only with `className`?